tests/unit/App1.spec.ts

```diff
@@ -0,0 +1,184 @@
+import Vue from 'vue'
```

Comment:
I broke up the previous 3 describe blocks into separate files since they seemed to run concurrently, which caused issues with store data (obviously).

Reply:
Breaking up the tests into 3 or 4 files made them all run concurrently, and there's a component in SbcHeader that doesn't like multiple copies of itself and caused unit test error.

So all App tests are now back into 1 file. Instead I fixed the data store issue 😊